matching-notification-api/src/main/java/org/phenotips/matchingnotification/notification/internal/DefaultUserPatientMatchEmail.java

```diff
@@ -128,9 +130,11 @@ protected void setTo()
     {
         super.setTo();
         try {
-            Address userEmail = getUserEmails();
-            if (userEmail != null) {
-                this.mimeMessage.addRecipient(RecipientType.CC, userEmail);
+            Set<Address> userEmails = getUserEmails();
+            for (Address email : userEmails) {
+                if (email != null) {
```

Comment:
`getUserEmails()` should ensure that no null objects are returned so that this null check is unnecessary.